addons/hr_holidays/tests/test_accrual_allocations.py

```diff
@@ -1145,6 +1145,7 @@ def test_accrual_leaves_taken_maximum(self):
                 'name': 'Accrual allocation for employee',
                 'accrual_plan_id': accrual_plan.id,
                 'employee_id': self.employee_emp.id,
+                'employee_ids': [Command.link(self.employee_emp_id)],
```

Comment:
I think this shows that ``employee_ids`` should be adapted accordingly when setting ``employee_id`` instead of adapting all the tests don't you think ?